contracts/vaults/YearnVaults/RibbonThetaYearnVault.sol

```diff
@@ -0,0 +1,379 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.3;
+pragma experimental ABIEncoderV2;
+
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "../../vendor/CustomSafeERC20.sol";
+import {SafeMath} from "@openzeppelin/contracts/math/SafeMath.sol";
+import {GnosisAuction} from "../../libraries/GnosisAuction.sol";
+import {
+    OptionsThetaYearnVaultStorage
+} from "../../storage/OptionsVaultYearnStorage.sol";
+import {Vault} from "../../libraries/Vault.sol";
+import {VaultLifecycleYearn} from "../../libraries/VaultLifecycleYearn.sol";
+import {ShareMath} from "../../libraries/ShareMath.sol";
+import {RibbonVault} from "./base/RibbonVault.sol";
+
+contract RibbonThetaYearnVault is RibbonVault, OptionsThetaYearnVaultStorage {
```

Comment:
We can remove `OptionsThetaYearnVaultStorage` since it's already inherited by RibbonVault?

Reply:
good catch. originally made sense because we had theta / delta vaults so they diverged in whats needed for storage, but we have deleted delta